content/departments/engineering/design/goals/index.md

```diff
@@ -0,0 +1,67 @@
+# Design team goals
+
+We have team-level design team goals to help us align and prioritize our
+focus as a team over the following 3-6 months.
+
+Our team-level goals cascade from the overall company [product vision and
```

Comment:
I think some clarification on the connection btw the company product vision/strategy and the (design team) themes would be helpful here! At first, I thought the themes were equivalent to the company product strategy, but it seems they're themes derived from the product strategy but more design team specific.